maidr/core/maidr.py

```diff
@@ -193,24 +193,36 @@ def generate_iframe_script(unique_id: str) -> str:
             resizing_script = f"""
                 function resizeIframe() {{
                     let iframe = document.getElementById('{unique_id}');
+
                     if (iframe && iframe.contentWindow && iframe.contentWindow.document) {{
                         let iframeDocument = iframe.contentWindow.document;
-                        let brailleContainer = iframeDocument.getElementById('braille-div');
+                        let brailleContainer = iframeDocument.getElementById('braille-input');
+
+                        iframe.style.height = 'auto';
```

Comment:
Greetings @dakshpokar,

I think we can set the height to auto in the `tags.iframe` construction. We set the height to 100% there and we are overriding it here. I removed this assignment and checked on Shiny by setting height to auto in `tags.iframe` construction - seems to work fine. Do check with Jupyter and share your thoughts on this.

Reply:
That's true I was going to add in the tags but the problem with that is that everytime the focus in and out script runs I want the original scrollable height of the content inside iframe. If we don't reset the previous height, the script was adding additional (not required) height. Hence, I just reset the height when the script runs so as to get the minimum scrollable height and then on the basis of that do the rest of the calculations.

Reply:
Sure, in that case this change is feasible.